src/components/image/Image.constants.ts

```diff
@@ -0,0 +1,13 @@
+import { TShapeBorderRadius } from '../../foundations/shape';
+
+import TImageSize from './Image.types';
+
+const DEFAULT_IMAGE_SIZE: TImageSize = 'auto';
+
+const DEFAULT_IMAGE_BORDER_RADIUS: TShapeBorderRadius = 4;
+
+const DEFAULT_IMAGE_SOURCE = 'http://placekitten.com/200/200';
```

Comment:
we should not set a default value. When there is not URL set fire a console.warn and return `null` from the component